### PR TITLE
New version: Ndd version 2.15.0

### PR DESCRIPTION
**Pull request by hjkl950217**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/152320)

### CODE REVIEW

File: manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - InstallerUrl: https://github.com/cxasm/notepad--/releases/download/notepad-v2.15/Notepad--v2.15.0-plugin-Installer.exe
+    InstallerLocale: en-US
+    Architecture: x64
+    InstallerType: inno
+    InstallerSha256: 322B7A07FB11227476DEFE6346DBA7871AA6B8A90C7411FE82B050E37C3E4F3F
+    Scope: machine
+    InstallerSwitches:
+      Custom: /ALLUSERS
+ManifestType: installer
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.installer.yaml:40]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Author: cxasm
+PackageName: Notepad--
+PackageUrl: https://gitee.com/cxasm/notepad--
+License: GPL-3.0 license
+# Copyright: https://gitee.com/cxasm/notepad--/blob/master/LICENSE
+CopyrightUrl: https://gitee.com/cxasm/notepad--/blob/master/LICENSE
+ShortDescription: A cross-platform alternative notepad application.
+ReleaseNotesUrl: https://gitee.com/cxasm/notepad--/releases/tag/v2.15
+ManifestType: defaultLocale
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.locale.zh-CN.yaml:20]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: ndd.Notepad--
+PackageVersion: 2.15.0
+DefaultLocale: zh-CN
+ManifestType: version
+ManifestVersion: 1.4.0
```
Comment (stephengillie) [inline manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.locale.zh-CN.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# PublisherUrl: https://gitee.com/cxasm/notepad--
+# PublisherSupportUrl: https://gitee.com/cxasm/notepad--
+# PrivacyUrl: https://gitee.com/cxasm/notepad--
+Author: cxasm
+PackageName: Notepad--
+PackageUrl: https://gitee.com/cxasm/notepad--
+License: GPL-3.0 license
+# Copyright: https://gitee.com/cxasm/notepad--/blob/master/LICENSE
+CopyrightUrl: https://gitee.com/cxasm/notepad--/blob/master/LICENSE
+ShortDescription: A cross-platform alternative notepad application.
```
Comment (stephengillie) [inline manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.locale.zh-CN.yaml:17]: Separately, should the `zh-CN` `ShortDescription` be possibly in Mandarin or Cantonese?

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @hjkl950217,

This manifest version has been depreciated, but most fields are identical to the newest version. In most cases, just increasing the ManifestVersion is enough to pass this check.

File: manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.installer.yaml (hunk 1 above)
Comment (hjkl950217) [reply manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.installer.yaml:40]: OK

File: manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.locale.zh-CN.yaml (hunk 4 above)
Comment (hjkl950217) [reply manifests/n/ndd/Notepad--/2.15.0/ndd.Notepad--.locale.zh-CN.yaml:17]: 你好像会中文，我可以和你用中文沟通吗？（You seem to speak Chinese. Can I communicate with you in Chinese?）

你是询问`ShortDescription`是什么语言吗？  不是太懂winget-paks规则，可以直接在这个`locale.zh-cn`下面写中文吗？  使用普通话(Are you asking what language `ShortDescription` is in? I don’t know the rules of winget-paks very well. Can I write Chinese directly under this `locale.zh-cn`? Use Mandarin)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**